assets/scss/components/_popover.scss

```diff
@@ -51,6 +51,7 @@ $view-minWidth: 320px;
 	& > span,
 	& > div,
 	& > button {
+		color: $C_textPrimary;
```

Comment:
I am using a button element, so this solved my issue, but what about anchor elements?

Reply:
Anchors are also in this selector. See line 50

Reply:
Doh, Github hid the top part of the rule. Looks good!